api/docker/caddy/Caddyfile

```diff
@@ -62,7 +51,6 @@
 			{$MERCURE_EXTRA_DIRECTIVES}
 		}
 		vulcain
-		push
```

Comment:
Vulcain is useless for us, according to the experiments I did last autumn. But push could still be useful, and thus we have so far kept mercure in. Do you have any insight on why this was removed or how it was replaced?

Reply:
HTTP/2 Server Push was disabled in Chrome 106. See https://developer.chrome.com/blog/removing-push/ and https://github.com/dunglas/vulcain/issues/70. So there's not much point in keeping this.

Mercure doesn't use HTTP/2 Server Push. It uses SSE (server side events).

Reply:
Oh, so this push directive was for vulcain, not mercure, I see.